frontend/src/components/pages/topics/Topic.List.tsx

```diff
@@ -164,87 +163,58 @@ class TopicList extends PageComponent {
-                                        </DeleteDisabledTooltip>
-                                    </div>
-                                ),
-                            },
-                        ]}
-                        search={{
-                            searchColumnIndex: 0,
-                            isRowMatch: (row, regex) => {
-                                if (regex.test(row.topicName)) return true;
-                                if (regex.test(row.cleanupPolicy)) return true;
-                                return false;
-                            },
-                        }}
-                        observableSettings={uiSettings.topicList}
-                        onRow={(record) => ({
-                            onClick: () =>
-                                appGlobal.history.push(`/topics/${encodeURIComponent(record.topicName)}`),
-                        })}
-                        rowClassName="hoverLink"
-                    />
+                                {
+                                    header: 'Partitions',
+                                    accessorKey: 'partitions',
+                                    cell: ({row: {original: topic}}) => topic.partitionCount,
+                                },
+                                {
+                                    header: 'Replicas',
+                                    accessorKey: 'replicationFactor',
+                                },
+                                {
+                                    header: 'CleanupPolicy',
+                                    accessorKey: 'cleanupPolicy',
+                                },
+                                {
+                                    header: 'Size',
+                                    accessorKey: 'size',
+                                    cell: ({row: {original: topic}}) => renderLogDirSummary(topic.logDirSummary),
+                                },
+                                {
+                                    id: 'action',
+                                    header: '',
+                                    cell: ({row: {original: record}}) => (
+                                        <Flex gap={1}>
+                                            <DeleteDisabledTooltip topic={record}>
+                                                <Button
+                                                    variant="ghost"
+                                                    className="iconButton"
+                                                    onClick={(event) => {
+                                                        event.stopPropagation();
+                                                        this.topicToDelete = record;
+                                                    }}
+                                                >
+                                                    <Icon as={HiOutlineTrash} fontSize="22px"/>
```

Comment:
I think we should create an issue because using `fontSize` does work, but ideally we should be using `boxSize`

Reply:
Ben, just curious: why box size instead of fontsize? I've been using fontsize all the time for icons. What is the difference?